frontend/src/pages/Dashboard/data/actions.ts

```diff
@@ -36,3 +36,18 @@ export const GetData = {
     payload: { error },
   }),
 };
+
+export const ChangeProfileIcon = {
+  request: (url: string) => ({
+    type: ActionTypes.CHANGE_PROFILE_ICON_REQUEST,
+    payload: { url },
+  }),
+  success: (url: string) => ({
+    type: ActionTypes.CHANGE_PROFILE_ICON_SUCCESS,
+    payload: { url },
+  }),
+  error: (error: Error) => ({
+    type: ActionTypes.CREATE_USER_ERROR,
```

Comment:
should this be `CHANGE_PROFILE_ICON_ERROR`?

Reply:
good catch lol

Reply:
also ya will change styling for mobile